addons/finger/functions/fnc_perFrameEH.sqf

```diff
@@ -34,6 +34,11 @@ private _iconSize = BASE_SIZE * 0.10713 * (call EFUNC(common,getZoom));
         // Fade out:
         _drawColor set [3, ((_drawColor select 3) * ((_timeLeftToShow min 0.5) / 0.5))];
 
+        private _iconSize = _iconBaseSize;
+        if (GVAR(proximityScaling)) then {
```

Comment:
I think we could get away with just a single setting?
```suggestion
        if (GVAR(sizeCoef) != 1) then {
```

Reply:
I want people to be able to set a custom scale if they want, and also to enable/disable distance based scaling

Reply:
nm I guess they do different things